README.md

```diff
@@ -28,6 +33,41 @@ Thanks for taking the time to join our community and start contributing!
 | `OPERATOR_SCANNER_AQUA_CSP_ENABLED`     | `false`              | The flag to enable Aqua CSP vulnerability scanner |
 | `OPERATOR_SCANNER_AQUA_CSP_VERSION`     | `5.0`                | The version of Aqua CSP scannercli container image to be used |
 
+## Install modes
+
+The values of the `OPERATOR_NAMESPACE` and `OPERATOR_TARGET_NAMESPACES` determine the install mode,
+which in turn determines the multitenancy support of the operator.
+
+| MODE            | OPERATOR_NAMESPACE | OPERATOR_TARGET_NAMESPACES | DESCRIPTION |
+| --------------- | ------------------ | -------------------------- | ----------- |
+| OwnNamespace    | `operators`        | `operators`                | The operator can be configured to watch events in the namespace it is deployed in. |
+| SingleNamespace | `operators`        | `foo`                      | The operator can be configured to watch for events in a single namespace that the operator is not deployed in. |
+| MultiNamespace  | `operators`        | `foo,bar,baz`              | The operator can be configured to watch for events in more than one namespace. |
+| AllNamespaces   | `operators`        | ""                         | The operator can be configured to watch for events in all namespaces. |
+
+## Vulnerability scanners
+
+To enable Aqua CSP as vulnerability scanner set the value of the `OPERATOR_SCANNER_AQUA_CSP_ENABLED` to `true` and
+disable the default Trivy scanner by setting `OPERATOR_SCANNER_TRIVY_ENABLED` to `false`.
```

Comment:
What happens if both are set to true? 
Eventually we might want to make this even more configurable so we can support other third party scanners (I think this will be sufficient for first release though)

Reply:
We check in code that there's only 1 active scanner enabled. If both are enable the operator will exit with error status and descriptive message.